src/go/rpk/pkg/cli/cmd/wasm/kafka_utils.go

```diff
@@ -0,0 +1,201 @@
+package transform
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"time"
+	"vectorized/pkg/cli/cmd/container/common"
+	"vectorized/pkg/config"
+	"vectorized/pkg/kafka"
+
+	"github.com/Shopify/sarama"
+	"github.com/burdiyan/kafkautil"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"golang.org/x/sync/errgroup"
+)
+
+const coprocessorTopic = "coprocessor_internal_topic"
+
+// Try to read the config from the default expected locations, or from the
+// specific path passed with --config. If --config wasn't passed, and the config
+// wasn't found, return the default configuration.
+func FindConfigFile(
```

Comment:
I thought the idea was to reuse these in the `api` command package too.

Reply:
No, we told about to create a new file and move the common functions to that file. did you prefer to keep all function on `api` module and `wasm` module use them

Reply:
Done, I create a new file and move all function related to Kafka configuration